ooniprobe/PrefixHeader.pch

```diff
@@ -21,4 +21,151 @@
 #define color_bad_red [UIColor colorWithRed:204.0/255.0 green:0.0/255.0 blue:0.0/255.0 alpha:1.0]
 #define color_warning_orange [UIColor colorWithRed:255.0/255.0 green:165.0/255.0 blue:0.0/255.0 alpha:1.0]
 
+#define color_base @"#0588CB"
+#define color_black @"#1f2528"
+#define color_gray @"#5b6c75"
+#define color_gray0 @"#f6f9f9"
+#define color_gray1 @"#e1e6e9"
+#define color_gray2 @"#c8d0d4"
+#define color_gray3 @"#aab7bf"
+#define color_gray4 @"#8699a3"
+#define color_gray5 @"#5b6c75"
+#define color_gray6 @"#516068"
+#define color_gray7 @"#455158"
+#define color_gray8 @"#364045"
+#define color_gray9 @"#202529"
+#define color_blue @"#0588cb"
+#define color_blue0 @"#e2f6ff"
+#define color_blue1 @"#bde9fe"
+#define color_blue2 @"#92d8fc"
+#define color_blue3 @"#5ac4fb"
+#define color_blue4 @"#0daaf9"
+#define color_blue5 @"#0588cb"
+#define color_blue6 @"#057db9"
+#define color_blue7 @"#0470a6"
+#define color_blue8 @"#046191"
+#define color_blue9 @"#034f77"
+#define color_indigo @"#0525cb"
+#define color_indigo0 @"#f0f2ff"
+#define color_indigo1 @"#d6dcfe"
+#define color_indigo2 @"#b6c2fe"
+#define color_indigo3 @"#92a3fc"
+#define color_indigo4 @"#5e77fb"
+#define color_indigo5 @"#0525cb"
+#define color_indigo6 @"#0526d0"
+#define color_indigo7 @"#0527d5"
+#define color_indigo8 @"#0528db"
+#define color_indigo9 @"#0528e0"
+#define color_violet @"#4805cb"
+#define color_violet0 @"#f5f1ff"
+#define color_violet1 @"#e5d7fe"
+#define color_violet2 @"#d2bafe"
+#define color_violet3 @"#b996fc"
+#define color_violet4 @"#9664fb"
+#define color_violet5 @"#4805cb"
+#define color_violet6 @"#4a05cf"
+#define color_violet7 @"#4b05d4"
+#define color_violet8 @"#4d05d9"
+#define color_violet9 @"#4e05dd"
+#define color_fuschia @"#ab05cb"
```

Comment:
Shouldn't this be fucsia?

Reply:
Yes that is true (the spelling should actually be fuschia, though). It's a mistyping of the color in the openobservatory/design repo: I filed this issue to track it: https://github.com/OpenObservatory/design/issues/24 (it may require changing some of the dependencies of the library as well).